src/server/pfs/fuse/server.go

```diff
@@ -1180,29 +1180,18 @@ func (m *MountStateMachine) RefreshMountState() error {
 	if err != nil {
 		return err
 	}
-
-	// set the latest commit on the branch in our LatestCommit
-	m.LatestCommit = branchInfo.Head.ID
-
-	// calculate how many commits behind LatestCommit ActualMountedCommit is
-	listClient, err := m.manager.Client.PfsAPIClient.ListCommit(m.manager.Client.Ctx(), &pfs.ListCommitRequest{
-		Repo: branchInfo.Branch.Repo,
-		To:   branchInfo.Head,
-		All:  true,
-	})
-	if err != nil {
-		return grpcutil.ScrubGRPC(err)
-	}
-	commitInfos, err := clientsdk.ListCommit(listClient)
+	commitInfos, err := m.manager.Client.ListCommit(branchInfo.Branch.Repo, branchInfo.Head, nil, 0)
 	if err != nil {
 		return err
 	}
+	m.LatestCommit = commitInfos[0].Commit.ID
+
 	// reverse slice
 	for i, j := 0, len(commitInfos)-1; i < j; i, j = i+1, j-1 {
 		commitInfos[i], commitInfos[j] = commitInfos[j], commitInfos[i]
 	}
 
-	// iterate over commits in branch, counting how many are behind LatestCommit
+	// iterate over non-alias commits in branch, calculating how many commits behind LatestCommit ActualMountedCommit is
```

Comment:
It looks like the comment changed, but the code didn't—just checking that that's intentional? This always skips alias commits?

Reply:
Yep it's intentional. It's to help clarify that the `ListCommit` rpc above returns non-alias commits by default.